app/Http/Controllers/Api/AssetModelsController.php

```diff
@@ -198,7 +198,8 @@ public function destroy($id)
      */
     public function selectlist(Request $request)
     {
-
+        $this->authorize('view', AssetModel::class);
```

Comment:
This will break other things throughout the system. If I have permission to edit an asset, for example, I need to be able to view the list of models, otherwise I'll never be able to save the asset, since the models list won't load. We document some of those considerations here: https://snipe-it.readme.io/docs/user-permissions#section-permissions-considerations

Reply:
Ahh, sorry. Should have RTFM. :books:

Reply:
It's okay - the permission dependencies are pretty complicated and affect things in ways that can be hard to predict unless you've screwed them in the past, as I have :) I made the exact same mistake when v4 came out.